bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -7773,6 +7773,35 @@ spec:
                               type: object
                             type: array
                         type: object
+                      secondaryNetworks:
+                        description: |-
+                          Define secondary networks to be checked for resources identification.
+                          In order to guarantee a correct identification, it is important that the indexed values form an unique identifier across the cluster.
+                          If there are collisions in the indexes (same index used by several resources), those resources might be wrongly labelled.
+                        items:
+                          properties:
+                            index:
+                              description: |-
+                                `index` is a list of fields to use for indexing the pods. They should form a unique Pod identifier across the cluster.
+                                Can be any of: MAC, IP, Interface
+                              items:
+                                description: 'Field to index for secondary network
+                                  pod identification, can be any of: MAC, IP, Interface'
+                                enum:
+                                - MAC
+                                - IP
+                                - Interface
+                                type: string
+                              type: array
+                            name:
+                              description: '`name` should match the network name as
+                                visible in the pods annotation ''k8s.v1.cni.cncf.io/network-status''.'
```

Comment:
```suggestion
                                visible in the pods annotation ''k8s.v1.cni.cncf.io/network-status'', typically specified as <namespace/network name>'
```

Reply:
I don't understand, where is `<namespace/network name>` used? Here we really want to tell: pick the `name` field from the `k8s.v1.cni.cncf.io/networks` annotation, there is no namespace involved?

Reply:
@jotak - when we configure `name: "my-vms/custom-nad"` in flowcollector the "my-vms" part is the namespace where custom-nad is deployed. After pod is **running** you see "my-vms/custom-nad"  in `k8s.v1.cni.cncf.io/networks ` annotation which includes namespace where custom-nad is configured. However at the time of Pod **configuration** k8s.v1.cni.cncf.io/networks annotation will look something like below, note here, name is simply "custom-nad"
annotations:
```
    k8s.v1.cni.cncf.io/networks: '[
      {
        "name": "custom-nad",
        "mac": "02:03:04:05:06:07",
        "interface": "myiface1",
        "ips": [
          "192.0.2.18/24"
          ]
      }
    ]'
```
It's just the minor detail that may help users avoid configuration mistake and they don't need to go back look at the Pod annotations **after** they're running.